src/web/mjs/connectors/YoungChampion.mjs

```diff
@@ -24,7 +24,7 @@ export default class YoungChampion extends Connector {
         this.queryMangaTitleURI = 'h1.series-h-title span:not([class])';
         this.queryManga = 'div.series-box-vertical a';
         this.queryMangaTitle = 'h2.title-text';
-        this.queryChapter = 'div.series-ep-list a';
+        this.queryChapter = 'div.series-ep-list a#null-false';
```

Comment:
This may affect other websites that are extending `YoungChampion`.
Have these websites been tested and are still working?

Reply:
Exactly, like ive said, there is only ComicEDU and Yanmaga, it goes without saying , i tested them.